src/app/shared-component/status-drawer.component.ts

```diff
@@ -10,7 +10,7 @@ import { WorkItem } from './../work-item/work-item';
 @Component({
 	selector   : 'status-drawer',
 	templateUrl: './status-drawer.component.html',
-	styleUrls  : ['src/app/shared-component/status-drawer.component.css'],
+	styleUrls  : ['/status-drawer.component.css'],
```

Comment:
@sanbornsen I didn't check functionality, but there was no 404 error in the client :) Now I am away from my system and I can only check your pull request very late today.